### PR TITLE
add ioctl stake2 register cmd

### PR DESCRIPTION
**Pull request by RuneAkko**
#2017

### CODE REVIEW

File: ioctl/cmd/action/stake2register.go [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+	gasPriceRau, err := gasPriceInRau()
+	if err != nil {
+		return output.NewError(0, "failed to get gas price", err)
+	}
+	nonce, err := nonce(sender)
+	if err != nil {
+		return output.NewError(0, "failed to get nonce ", err)
+	}
+	cr, err := action.NewCandidateRegister(nonce, name, operatorAddrStr, rewardAddrStr, ownerAddrStr, amountStringInRau, duration, autoRestake, payload, gasLimit, gasPriceRau)
```
Comment (dustinxie) [inline ioctl/cmd/action/stake2register.go:113]: can directly use amountInRau.String() here

Comment (dustinxie) [reply ioctl/cmd/action/stake2register.go:113]: need to use stake2AutoRestake here, right?

Comment (RuneAkko) [reply ioctl/cmd/action/stake2register.go:113]: sure

Comment (RuneAkko) [reply ioctl/cmd/action/stake2register.go]: other PRs don't have same issue.

File: ioctl/cmd/action/stake2register.go [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+	if err != nil {
+		return output.NewError(output.AddressError, "failed to get reward address", err)
+	}
+	ownerAddrStr, err := util.Address(args[3])
+	if err != nil {
+		return output.NewError(output.AddressError, "failed to get owner address", err)
+	}
+
+	amountStr := args[4]
+	amountInRau, err := util.StringToRau(amountStr, util.IotxDecimalNum)
```
Comment (dustinxie) [inline ioctl/cmd/action/stake2register.go:76]: directly use args[4]

**Pull request merged**